code/__HELPERS/priority_announce.dm

```diff
@@ -27,6 +29,11 @@
 				GLOB.news_network.SubmitArticle(title + "<br><br>" + text, "Central Command", "Station Announcements", null)
 
 	announcement += "<br><span class='alert'>[html_encode(text)]</span><br>"
```

Comment:
```suggestion
```
found the broken line

Reply:
fully omit it, it's done 2 lines below, hence double announcements

Reply:
Damn byodn code!!